plugins/servicenow/icon_servicenow/actions/get_security_incident/action.py

```diff
@@ -16,14 +18,15 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        return {
-            Output.SECURITY_INCIDENT: convert_security_incident_fields(
-                return_non_empty(
-                    self.connection.request.make_request(
-                        endpoint=f"{self.connection.security_incident_url}/{params.get(Input.SYS_ID)}", method="GET"
-                    )
-                    .get("resource", {})
-                    .get("result", {})
-                )
-            )
-        }
+
+        response = self.connection.request.make_request(
+            endpoint=f"{self.connection.security_incident_url}/{params.get(Input.SYS_ID)}", method="GET"
+        )
+        try:
+            result = response.get("resource", {}).get("result", {})
```

Comment:
It won't throw `KeyError` as you are obtaining the values using `get()` instead of directly by key.
```suggestion
            result = response["resource"].get("result", {})
```

Reply:
Thanks, have standardised how all actions access dictionary variables.